spec/tasks/pages_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
```

Comment:
LOL I shouldn't say this but I feel like we should get rid of this test if it's too costly or fix its present state.

Hear me out:

- this test is testing a task that should only be enabled in development (forgot to flag that we should make sure that the task doesn't do anything in production mode at least).
- this test spawns threads which are going to change data on disk, so if the user hits Ctrl-C between before and after but the thread finishes, they'll have changes on disk

If we want to keep this test in, we should either:

- use Tempfile as the destination (you could copy the fixture file to a temp file and then unleash the threads on that)
- mock `File.read`/`File.write` to write in a memory mapped file but this might be trickier

Reply:
Makes a lot of sense! I agree that we should avoid the possible side effects the test can have.

Just sent in the fix for this.

Reply:
I'm with @rhymes, I really think we should skip the test for this.  There are lots of sleeps in there which are going to be slow and its not code we rely on in production so if it breaks its not the end of the world.